pkg/s3utils/utils.go

```diff
@@ -84,6 +84,9 @@ func IsVirtualHostSupported(endpointURL url.URL, bucketName string) bool {
 // AmazonS3Host - regular expression used to determine if an arg is s3 host.
 var AmazonS3Host = regexp.MustCompile("^s3[.-]?(.*?)\\.amazonaws\\.com$")
 
+// AmazonS3ChinaHost - regular expression used to determin if the arg is s3 china host.
+var AmazonS3ChinaHost = regexp.MustCompile("^s3[.-]?(.*?)\\.amazonaws\\.com.cn$")
```

Comment:
doesnt the . before cn$ need to be escaped as well?

Reply:
typo -> determin

Reply:
> doesnt the . before cn$ need to be escaped as well?

oh yes @poornas